### ISSUE TITLE
FAQ dark theme answer seems to be wrong

### ISSUE THREAD

**Issue by hippietrail** [NationalSecurityAgency/ghidra#6948]
**Describe the bug**
The answer to the FAQ ["Does Ghidra have a dark theme?"](https://github.com/NationalSecurityAgency/ghidra/wiki/Frequently-asked-questions#does-ghidra-have-a-dark-theme) says:

Ghidra 10.3+ provides a dark theme out-of-the-box. To enable this theme, from the Front-End GUI, click Edit → Theme → Switch... → Flat Dark Theme.

But there is no "Theme" entry when I click "Edit" on the Front-End GUI:
<img width="279" alt="image" src="https://github.com/user-attachments/assets/36b7d3c6-a375-4f3d-bf4c-a994578281e1">

**Environment (please complete the following information):**
 - OS: macOS 14.4 (23E214)
 - Java Version: Java 8 Update 421
 - Ghidra Version: 11.1.2
 - Ghidra Origin: official GitHub distro

**Comment by ryanmkurtz** [comment]
Can you please confirm your version of Ghidra by doing `Help -> About Ghidra`?

**Comment by ryanmkurtz** [comment]
Also, that action is controlled by the `ThemeManagerPlugin`.  From the Front End GUI, click `File -> Configure`.  Then click `Configure` under the first `Utility` group.  You should see a checkbox for the `ThemeManagerPlugin` in the popup.

**Comment by hippietrail** [comment]
> Also, that action is controlled by the `ThemeManagerPlugin`. From the Front End GUI, click `File -> Configure`. Then click `Configure` under the first `Utility` group. You should see a checkbox for the `ThemeManagerPlugin` in the popup.

Ah OK this fixes it. I think some of the plugins etc sometimes go missing when I get certain kinds of errors quitting Ghidra while working on my Java Loader extensions under Eclipse, and I don't always notice all of the things that break when it happens.

**Comment by ryanmkurtz** [comment]
I've seen the plugins in the CodeBrowser tool get messed up, but I don't think i've ever seen them get messed up in the Front End.

**Comment by hippietrail** [comment]
> I've seen the plugins in the CodeBrowser tool get messed up, but I don't think i've ever seen them get messed up in the Front End.

Yes I have to fix the CodeBrowser every now and then. I got help with that here. I also lost all my hotkeys/shortcuts inside the CodeBrowser once. Usually I don't use most of the other features so don't notice other things going awry. I've never attempted to change themes until now.
